minecraft-mohist/minecraft-mohist.json

```diff
@@ -0,0 +1,98 @@
+{
+    "type": "minecraft-java",
+    "display": "Mohist - Minecraft",
+    "name": "minecraft-mohist",
+    "install": [
+        {
+            "files": [
+                "https://mohistmc.com/builds/${version}/mohist-${version}-${build}-server.jar"
+            ],
+            "type": "download"
+        },
+        {
+            "source": "mohist-*.jar",
+            "target": "mohist.jar",
+            "type": "move"
+        },
+        {
+            "target": "eula.txt",
+            "text": "eula=${eula}",
+            "type": "writefile"
+        },
+        {
+            "target": "server.properties",
+            "text": "server-ip=${ip}\nserver-port=${port}\nmotd=${motd}",
+            "type": "writefile"
+        }
+    ],
+    "run": {
+        "command": "java -Xmx${memory}M -Djline.terminal=jline.UnsupportedTerminal -jar mohist.jar",
```

Comment:
due to changes in the newest version of pufferpanel, you'll probably need to add flags to make it not spam extra symbols into the console, for paper these are `-Dterminal.jline=false` and `-Dterminal.ansi=true`, i don't know mohist though, so it might need different flags (or maybe even none at all, should it neither do colored logging nor use jline for input management)